jscomp/syntax/src/res_printer.ml

```diff
@@ -4730,6 +4730,20 @@ and print_arguments ~state ?(partial = false)
     in
     Doc.concat [Doc.lparen; arg_doc; Doc.rparen]
   | args ->
+    (* Avoid printing trailing comma when there is ... in function application *)
```

Comment:
I don't think it matters whether to check if `...` happens at the end because that's already considered as syntax error.